src/runtimes/node/src_files.js

```diff
@@ -30,7 +30,7 @@ const getPathsOfIncludedFiles = async (includedFiles, basePath) => {
     { include: [], exclude: [] },
   )
   const pathGroups = await Promise.all(
-    include.map((expression) => pGlob(expression, { absolute: true, cwd: basePath, ignore: exclude })),
+    include.map((expression) => pGlob(expression, { absolute: true, cwd: basePath, ignore: exclude, nodir: true })),
```

Comment:
This ignores directories, which in practice means that a directory will only be included if it has files inside it.

For example, imagine:

```
- full
    - file.js
- empty
```

The glob `**` without `nodir: true` would yield `full`, `full/file.js` and `empty`. We can't copy directories when using `archiveFormat: "none"`, so it'd throw an error when handling `full` and `empty`.

By using `nodir: true`, the glob would yield simply `full/file.js`. Upon copying, it would be placed in the right location (i.e. any parent directories would be created automatically).